.github/actions/setup-build/action.yaml

```diff
@@ -0,0 +1,110 @@
+name: setup-build-env
```

Comment:
we need to adjust the commit message. other than that, LGTM.

Reply:
thanks for pointing this out. i updated the cover letter, but forgot to update the commit message.